src/storage/disk_table.h

```diff
@@ -350,6 +348,8 @@ class DiskTable : public Table {
     inline uint64_t GetRecordByteSize() const override { return 0; }
     uint64_t GetRecordIdxByteSize() override;
 
+    int GetCount(uint32_t index, const std::string& pk, uint64_t& count);
```

Comment:
since both `MemTable` and `DiskTable` have `GetCount`, move it to `Table` and override